src/moscot/problems/space/_mapping.py

```diff
@@ -222,15 +239,15 @@ def prepare(
 
     def solve(
         self,
-        alpha: float = 0.5,
+        alpha: Optional[float] = 0.5,
```

Comment:
Why is alpha optional here? Optional means Union[float, None] but I am not sure we want that. Looking at the code it isn't handled when alpha=None so I am assuming this is a mistake.

Reply:
I agree! I think we should stick to passing alpha=0 for linear problem @Marius1311

Reply:
I find it a bit unintuitive that when you prepare a problem without quadratic terms,  you still need to explicitly pass `alpha=0` in the solve method, that's why I thought it should be optional in that case.